Dockerfile

```
@@ -0,0 +1,31 @@
+FROM ubuntu
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)